WalletWasabi.Gui/Icons/Icons.xaml

```diff
@@ -216,6 +216,48 @@
+          <GeometryDrawing Brush="#FF29292C" Geometry="m 0 0 v 1 2.9960938 2.0019531 1 V 9 12 H 9 V 9 6.9980469 3.9960938 0 Z M 2 2 H 7 V 3.9960938 6.9980469 9 10 H 2 v -1 -2.0019531 -1 -2.0019531 z"/>
+          <GeometryDrawing Brush="#FF3B3B3B" Geometry="m 1.2293781 1.3284036 v 2.741135 1.8315928 0.9149029 1.8315927 1.829805 H 7.633699 V 8.647627 6.8160343 4.0695386 1.3284036 Z"/>
+          <GeometryDrawing Brush="#FFC4C4C4" Geometry="m 0.9992292 0.9996195 v 2.9960938 2.0019531 1 2.001953 1.9999996 H 7.999229 V 8.9996194 6.9976664 3.9957133 0.9996195 Z m 1 1 H 6.999229 v 1.9960938 0.00195 H 1.9992292 v -0.00195 z m 0 2.9980469 h 1 v 1 h -1 z m 1.9999998 0 h 1 v 1 h -1 z m 2 0 h 1 v 1 h -1 z m -3.9999998 2.001953 h 1 v 1 h -1 z m 1.9999998 0 h 1 v 1 h -1 z m 2 0 h 1 v 1 h -1 z m -3.9999998 2 h 1 v 1 h -1 z m 1.9999998 0 h 1 v 1 h -1 z m 2 0 h 1 v 1 h -1 z"/>
+          <GeometryDrawing Brush="#FF29292C" Geometry="m 11.50786 6.9109763 c -1.416666 0 -2.6064413 0.567256 -3.3730473 1.429688 -0.766606 0.862432 -1.126953 1.9730897 -1.126953 3.0703117 0 1.097222 0.360347 2.207881 1.126953 3.070312 0.766606 0.862432 1.9563813 1.429688 3.3730473 1.429688 1.416666 0 2.606441 -0.567256 3.373047 -1.429688 0.766606 -0.862431 1.126953 -1.97309 1.126953 -3.070312 0 -1.097222 -0.360347 -2.2078797 -1.126953 -3.0703117 -0.766606 -0.862432 -1.956381 -1.429688 -3.373047 -1.429688 z"/>
+          <GeometryDrawing Brush="#FFC4C4C4" Geometry="m 11.50786 7.9109763 c -4.6666653 0 -4.6666653 6.9999997 0 6.9999997 4.666665 0 4.666665 -6.9999997 0 -6.9999997 z"/>
+        </DrawingGroup.Children>
+      </DrawingGroup>
+
+      <DrawingGroup x:Key="WalletExplorer_WatchOnlyClosedWallet">
+        <DrawingGroup.Children>
+          <GeometryDrawing Brush="#00FFFFFF" Geometry="F1M16,16L0,16 0,0 16,0z" />
+          <GeometryDrawing Brush="#FF252526" Geometry="F1M16,7L16,9.5 15,11C14.86,12.676 13.341,14 11.5,14 9.659,14 8.14,12.676 8,11L7.973,11C7.833,12.676 6.341,14 4.5,14 2.659,14 1.14,12.676 1,11L0,9.5 0,7 1,7 7,1.001 8.999,1 15,7z" />
+          <GeometryDrawing Brush="#FFC4C4C4" Geometry="F1M11.5,12C10.672,12 10,11.328 10,10.5 10,9.672 10.672,9 11.5,9 12.328,9 13,9.672 13,10.5 13,11.328 12.328,12 11.5,12 M4.5,12C3.672,12 3,11.328 3,10.5 3,9.672 3.672,9 4.5,9 5.328,9 6,9.672 6,10.5 6,11.328 5.328,12 4.5,12 M14,8L9,3 9,4 12,8 9,8C8.5,8.5 8,8.5 8,8.5 7.5,8.5 7,8 7,8L4,8 7,4 7,3 2,8 1,8 1,9 2,10.5C2,11.881 3.119,13 4.5,13 5.881,13 7,11.881 7,10.5 7.5,10 8,10 8,10 8.5,10 9,10.5 9,10.5 9,11.881 10.119,13 11.5,13 12.881,13 14,11.881 14,10.5L15,9 15,8z" />
+          <GeometryDrawing Brush="#FF29292C" Geometry="F1M11.5,9C10.672,9 10,9.672 10,10.5 10,11.328 10.672,12 11.5,12 12.328,12 13,11.328 13,10.5 13,9.672 12.328,9 11.5,9 M6,10.5C6,11.328 5.328,12 4.5,12 3.672,12 3,11.328 3,10.5 3,9.672 3.672,9 4.5,9 5.328,9 6,9.672 6,10.5" />
+        </DrawingGroup.Children>
+      </DrawingGroup>
+
+      <DrawingGroup x:Key="WalletExplorer_WatchOnlyOpenWallet">
```

Comment:
Maybe rename this to `WalletExplorer_OpenWatchOnlyWallet`?

Reply:
That sounds like an action, the icon has to reflect the state of the wallet. I think it is okay as it is.

Reply:
I was suggesting that just to follow what we already have: `WalletExplorer_OpenHardwareWallet`.

We can maybe rename them to something like
`WalletExplorer_HardwareWalletOpen`
`WalletExplorer_HardwareWalletClosed`
`WalletExplorer_WatchOnlyWalletOpen`
`WalletExplorer_WatchOnlyWalletClosed`